File: Java/프로그래머스/0/181895. 배열 만들기 3/배열 만들기 3.java
class Solution {
    public int[] solution(int[] arr, int[][] intervals) {
        int[] answer = new int[(intervals[0][1]-intervals[0][0])+(intervals[1][1]-intervals[1][0])+2];
        int n=0;
        int m=0;
        for(int i=intervals[0][0]; i<=intervals[0][1];i++){
            answer[m]=arr[i];
            m++;
        }
        for(int i=intervals[1][0]; i<=intervals[1][1];i++){
            answer[m]=arr[i];
            m++;
        }
        return answer;
    }
}

File: 프로그래머스/0/181895. 배열 만들기 3/배열 만들기 3.js
function solution(arr, intervals) {
    var answer = [];
    var answer = [,,];
    let n=0;
    let m=0;
    for(let i=0; i<intervals.length;i++){
        for(let j=intervals[i][0]; j<=intervals[i][1];j++){
            answer[m]=arr[j];
            m++;
        }
    }
    return answer;
}